### PR TITLE
:paperclip: Replacing CodePlex with GitHub Per #2974

### PR DESCRIPTION
**Pull request by isaacrlevin**
Final changes to remove all references to CodePlex in dotnet docs. All links pointed to GitHub or other docs

### CODE REVIEW

**Comment by isaac2004** [comment]
@mairaw please review

**Comment by mairaw** [comment]
Thank you @isaac2004. I or someone from my team will get to this as soon as possible. 

Related to issue #2974

File: xml/System.ComponentModel.DataAnnotations/ScaffoldColumnAttribute.xml [hunk 1]
```diff
@@ -89,11 +90,12 @@ End Class
   
 -   Visual Studio 2008 Service Pack 1 or Visual Developer 2008 Express Edition Service Pack 1.
```
Comment (rpetrusha) [inline xml/System.ComponentModel.DataAnnotations/ScaffoldColumnAttribute.xml]: The Example section should be commented out as previously.

File: xml/System.ComponentModel.DataAnnotations/ScaffoldTableAttribute.xml [hunk 2]
```diff
@@ -73,11 +74,12 @@ End Class
   
 -   Visual Studio 2008 Service Pack 1 or Visual Developer 2008 Express Edition Service Pack 1.
```
Comment (rpetrusha) [inline xml/System.ComponentModel.DataAnnotations/ScaffoldTableAttribute.xml]: Again, the entire Example section should be commented out as previously.

File: xml/System.Dynamic/DynamicMetaObject.xml [hunk 3]
```diff
@@ -26,12 +26,14 @@
   <Docs>
     <summary>Represents the dynamic binding and a binding logic of an object participating in the dynamic binding.</summary>
     <remarks>
-      <format type="text/markdown"><![CDATA[  
+      <format type="text/markdown">
+        <![CDATA[  
   
 ## Remarks  
- For more information, see documentation on the [CodePlex](http://go.microsoft.com/fwlink/?LinkId=141028) Web site and the MSDN article [Implementing Dynamic Interfaces](http://go.microsoft.com/fwlink/?LinkID=198161).  
+ For more information, see the MSDN article [Implementing Dynamic Interfaces](http://go.microsoft.com/fwlink/?LinkID=198161).
```
Comment (rpetrusha) [inline xml/System.Dynamic/DynamicMetaObject.xml]: For more information, download \[Implementing Dynamic Interfaces](http://download.microsoft.com/download/5/4/B/54B83DFE-D7AA-4155-9687-B0CF58FF65D7/implementing-dynamic-interfaces.pdf) in PDF format.

File: xml/System.Dynamic/DynamicObject.xml [hunk 4]
```diff
@@ -935,7 +1166,7 @@ Dim testExplicit2 = CTypeDynamic(sampleObject, GetType(String))
   
  You can get information about the type of the unary operation by using the `Operation` property of the `binder` parameter.  
   
- If your dynamic object is used only in C# and Visual Basic, the `binder.Operation` property can have one of the following values from the <xref:System.Linq.Expressions.ExpressionType> enumeration. However, in other languages such as IronPython or IronRuby, you can have other values. For more information about <xref:System.Linq.Expressions.ExpressionType>, see documentation on the [CodePlex](http://go.microsoft.com/fwlink/?LinkId=141028) Web site.  
+ If your dynamic object is used only in C# and Visual Basic, the `binder.Operation` property can have one of the following values from the <xref:System.Linq.Expressions.ExpressionType> enumeration. However, in other languages such as IronPython or IronRuby, you can have other values. For more information about <xref:System.Linq.Expressions.ExpressionType>, see [documentation](~/dotnet/api/system.linq.expressions.expressiontype).
```
Comment (rpetrusha) [inline xml/System.Dynamic/DynamicObject.xml]: There are two links to the same topic. Change this to, "For more information, see \<xref::System.Linq.Expressions.ExpressionType>."

File: xml/System.Dynamic/IDynamicMetaObjectProvider.xml [hunk 5]
```diff
@@ -23,14 +23,16 @@
...
     <remarks>
-      <format type="text/markdown"><![CDATA[  
+      <format type="text/markdown">
+        <![CDATA[  
   
 ## Remarks  
  Objects that want to participate in the binding process should implement an IDynamicMetaObjectProvider interface, and implement <xref:System.Dynamic.IDynamicMetaObjectProvider.GetMetaObject%2A> to return a <xref:System.Dynamic.DynamicMetaObject>.  
   
- For more information, see documentation on the [CodePlex](http://go.microsoft.com/fwlink/?LinkId=141028) Web site and the MSDN article [Implementing Dynamic Interfaces](http://go.microsoft.com/fwlink/?LinkID=198161).  
+ For more information, see documentation on the MSDN article [Implementing Dynamic Interfaces](http://go.microsoft.com/fwlink/?LinkID=198161).
```
Comment (rpetrusha) [inline xml/System.Dynamic/IDynamicMetaObjectProvider.xml]: This should be replaced with the language and link for the PDF used previously.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
I'm sorry this has taken so long, @isaac2004. I've added a number of comments.

**Comment by rpetrusha** [comment]
@cartertmp, could you take a look at the DLR-related topic here. We're trying to eliminated CodePlex references, and there don't appear to be any replacements for the DLR documentation/code that were available in CodePlex.

**Comment by isaac2004** [comment]
@rpetrusha probably going to be a bit before I can make these changes. Ha e some happy personal things going on. I will try next week sometime.

**Comment by isaac2004** [comment]
@rpetrusha changes done

File: docs/framework/add-ins/index.md [hunk 6]
```diff
@@ -39,7 +39,7 @@ manager: "wpickett"
 -   [Reference](#reference)  
   
 > [!NOTE]
->  You can find additional sample code, and customer technology previews of tools for building add-in pipelines, at the [Managed Extensibility and Add-In Framework site on CodePlex](http://go.microsoft.com/fwlink/?LinkId=121190).  
+>  For an example of building an add-in pipeline, see[Creating an Extensible Application](~/dotnet/framework/add-ins/walkthrough-create-extensible-app).
```
Comment (svick) [inline docs/framework/add-ins/index.md]: Missing space after "see".

File: xml/System.ComponentModel.DataAnnotations/DisplayFormatAttribute.xml [hunk 7]
```diff
@@ -101,11 +107,19 @@
...
-        <summary>Gets or sets a value that indicates whether the formatting string that is specified by the <see cref="P:System.ComponentModel.DataAnnotations.DisplayFormatAttribute.DataFormatString" /> property is applied to the field value when the data field is in edit mode.</summary>
+        <summary>Gets or sets a value that indicates whether the formatting string that is specified by the           
+          <see cref="P:System.ComponentModel.DataAnnotations.DisplayFormatAttribute.DataFormatString" /> property is applied to the field value when the data field is in edit mode.
+        
+        </summary>
         <value>
-          <see langword="true" /> if the formatting string applies to the field value in edit mode; otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
+          <see langword="true" /> if the formatting string applies to the field value in edit mode; otherwise,           
+          <see langword="false" />. The default is           
+          <see langword="false" />.
```
Comment (svick) [inline xml/System.ComponentModel.DataAnnotations/DisplayFormatAttribute.xml]: Is there a reason to reformat this code like this? Similar cases are also below.

Comment (isaac2004) [reply xml/System.ComponentModel.DataAnnotations/DisplayFormatAttribute.xml]: I am guessing it is a vscode autoformat thing. I will fix them.

**Comment by mairaw** [comment]
@isaac2004 and I chatted via Twitter and I've offered to help.

@isaac2004 @rpetrusha here's what I did. I restored some of the xml files impacted by this PR. I didn't comment out the samples. I just left the todo comment for us to review later but kept the examples and the links to codeplex. Codeplex will become read-only on Nov 6, so those specific links should continue to work.

I sent an email to the team asking what's the recommended database for samples now. My guess is that the answer will be one that was launched with SQL Server 2016 called WWI. If so, we should create an issue that will become a bigger project to convert samples that use SQL Server to use the new samples database.

Was there any other pending issue besides the loss of formatting in some of those xml files @rpetrusha?

Thanks!

**Review by mairaw — COMMENTED** [review]
A few more comments. In some cases, I'm really torn if we should just keep the link and not touch the topic given that the change is bigger than just removing CodePlex from the text.

**Comment by isaac2004** [comment]
@mairaw changes complete

**Comment by mairaw** [comment]
Hi @isaac2004, I'll take a look as soon as possible. Thank you!

File: xml/System.ComponentModel.DataAnnotations/ScaffoldTableAttribute.xml (hunk 2 above)
Comment (mairaw) [reply xml/System.ComponentModel.DataAnnotations/ScaffoldTableAttribute.xml]: I think the examples might still work. I'm leaving them for now.

**Review by mairaw — APPROVED** [review]

**Comment by mairaw** [comment]
@isaac2004 @rpetrusha I went through the PR and applied the last changes I thought were necessary. In some cases, I decided to revert the change and leave the codeplex link for now. I think some of those topics don’t even make sense anymore and could probably be retired from the current set. Let’s merge these changes once it builds successfully and we’ll see what to do with what’s left later on. Thanks Isaac!

**Comment by rpetrusha** [comment]
Thanks, @isaac2004! The build has completed, so we'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**